ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -1860,6 +1958,123 @@ seriesType<Highcharts.PackedBubbleSeries>(
                 );
             }
             return Point.prototype.destroy.apply(this, arguments as any);
+        },
+        firePointEvent: function (
+            this: Highcharts.PackedBubblePoint,
+            eventType: string,
+            eventArgs?: (Highcharts.Dictionary<any>|Event),
+            defaultFunction?: (
+                Highcharts.EventCallbackFunction<Highcharts.Point>|Function
+            )
+        ): void {
+            const point = this,
+                series = this.series,
+                seriesOptions = series.options;
+
+            // load event handlers on demand to save time on mouseover/out
+            if (this.isParentNode) {
+                if ((seriesOptions.point as any).events[eventType] ||
+                (
+                    point.options &&
+                    point.options.events &&
+                    (point.options.events as any)[eventType]
+                )
+                ) {
+                    point.importEvents();
+                }
+
+                // add default handler if in selection mode
+                if (
+                    eventType === 'click' &&
+                    seriesOptions.parentNode &&
+                    seriesOptions.parentNode.allowPointSelect
+                ) {
+                    defaultFunction = function (event: MouseEvent): void {
+                        // Control key is for Windows, meta (= Cmd key) for Mac,
+                        // Shift for Opera.
+                        if (point.select) { // #2911
+                            point.select(
+                                null as any,
+                                event.ctrlKey || event.metaKey || event.shiftKey
+                            );
+                        }
+                    };
+                }
+
+                fireEvent(point, eventType, eventArgs, defaultFunction);
+            } else {
+                return Point.prototype.firePointEvent.apply(this, arguments as any);
+            }
+        },
+        select: function (
```

Comment:
This method looks like a copy&paste of `point.select()`. How about instead doing something like this:

```
if (isParent) {
  this.series.getSelectedPoints = this.getSelectedParents;
  Point.proto.select(this, args);
  this.series.getSelectedPoints = H.Series.proto.getSelectedPoints;
} else {
  Point.proto.select(this, args);
}
```

Reply:
The same applies to the method above `firePointEvent` - can we reuse method instead of copy&paste, and just modify options that are used inside the method?